### DISCUSSION TITLE
[Feature] ability to add pictures to album automatically

### DISCUSSION THREAD

**Discussion by EdeAbreu23** [immich-app/immich#12036] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

so on google photos, there is an option that when creating an album, you can choose faces to add into the album and every time a picture match that face/faces, it will add that picture to the album..

for this to work, first when searching pictures with faces, you can already select multiple faces but the problem with that is that it will find every picture that matches all those faces even if there is more faces on that picture.. on google photos, there was an option that when selecting the first face, it will tell you if you just one that face alone or those faces only if you select it, it will search for picture that only match the face/faces..

so the way i was using that was, i created an album with my main family (Me, Wife and 3 kids) and every times those 5 faces were on a picture alone, it will add it to the album and then i will have a slideshow that will show those so i did not have to worry about keep adding more and more pictures

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1673
